pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: tiled
-version: 0.8.3
+version: 0.8.4
```

Comment:
This should normally not be bumped in this PR, but since I'll do a release after this is merged it doesn't matter this time.